src/Components/Modals/ModalPhotoDisplay/ModalPhotoDisplay.js

```diff
@@ -32,7 +33,8 @@ class ModalPhotoDisplay extends Component {
     }
 
     handleClickedArrow(isLeft){
-        const currentImageIndex = this.props.images.indexOf(this.props.currentClickedImage);
+        const currentImageSrc = this.props.currentClickedImage.split("/")[4];
```

Comment:
4 is a magic number (a number with no explanation). Better to give it a name and then use.